common.blocks/select/_mode/select_mode_radio.bemhtml

```diff
@@ -1,12 +1,12 @@
 block('select').mod('mode', 'radio')(
-    def().match(this._checkedOptions)(function() {
+    def().match(function() { return this._checkedOptions; })(function() {
         var checkedOptions = this._checkedOptions,
             firstOption = this._firstOption;
         if(firstOption && !checkedOptions.length) {
             firstOption.checked = true;
             checkedOptions = [firstOption];
         }
-        applyNext({ _checkedOption : checkedOptions[0] });
+        return applyNext({ _checkedOption : checkedOptions[0] });
     }),
 
     content()(function() {
```

Comment:
looks like it is defined at https://github.com/bem/bem-components/blob/feature/prepare-for-bem-xjst-v2/common.blocks/select/select.bemhtml#L34 and undefined in `block('select').mod('mode', 'radio')`